pkg/bundle/client.go

```diff
@@ -155,6 +160,41 @@ func (bc *bundleClient) CreateClusterNamespace(ctx context.Context, clusterName
 	return nil
 }
 
+func (bc *bundleClient) CreateClusterConfigMap(ctx context.Context, clusterName string) error {
+	name := auth.ConfigMapName
+	namespace := api.PackageNamespace + "-" + clusterName
+	key := types.NamespacedName{
+		Name:      name,
+		Namespace: namespace,
+	}
+
+	cm := &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+	}
+	err := bc.Get(ctx, key, cm)
+	if err == nil {
+		return nil
+	}
+
+	if !apierrors.IsNotFound(err) {
+		return err
+	}
+
+	cm.Data = make(map[string]string)
+	cm.Data[namespace] = "eksa-package-controller"
+	// Unfortunate workaround for emissary webhooks hard coded crd namespace
+	cm.Data["emissary-system"] = "eksa-package-placeholder"
```

Comment:
Did we do a check on existing packages and packages that will go out in this release, and confirm emissary is the only one that needs this workaround?

Reply:
New Packages so far do not create any namespaces. Exception was cert-manager however there is a patch for that.